src/main/java/frc/robot/subsystems/shooter/Shooter.java

```diff
@@ -0,0 +1,157 @@
+    private WPI_TalonFX rightTalon;
+    private boolean isTuning;
+
+    public Shooter() {
+        //setting up the talon
+        leftTalon = new WPI_TalonFX(robotConstants.can.LEFT_SHOOTER_TALON_FX);
+        leftTalon.setNeutralMode(NeutralMode.Coast);
+        leftTalon.configSelectedFeedbackSensor(TalonFXFeedbackDevice.IntegratedSensor, 0, 0);
+        leftTalon.config_kP(0, robotConstants.controlConstants.leftShooterSettings.getKP());
+        leftTalon.config_kI(0, robotConstants.controlConstants.leftShooterSettings.getKI());
+        leftTalon.config_kD(0, robotConstants.controlConstants.leftShooterSettings.getKD());
+        leftTalon.config_kF(0, robotConstants.shooterConstants.LEFT_KF);
+        leftTalon.selectProfileSlot(0, 0);
+        leftTalon.setInverted(robotConstants.shooterConstants.IS_LEFT_MOTOR_INVERTED);
+        leftTalon.setSensorPhase(robotConstants.shooterConstants.IS_LEFT_ENCODER_INVERTED);
+
+        rightTalon = new WPI_TalonFX(robotConstants.can.RIGHT_SHOOTER_TALON_FX);
+        rightTalon.setNeutralMode(NeutralMode.Coast);
+        rightTalon.configSelectedFeedbackSensor(TalonFXFeedbackDevice.IntegratedSensor, 0, 0);
+        rightTalon.config_kP(0, robotConstants.controlConstants.rightShooterSettings.getKP());
+        rightTalon.config_kI(0, robotConstants.controlConstants.rightShooterSettings.getKI());
+        rightTalon.config_kD(0, robotConstants.controlConstants.rightShooterSettings.getKD());
+        rightTalon.config_kF(0, robotConstants.shooterConstants.RIGHT_KF);
+        rightTalon.selectProfileSlot(0, 0);
+        rightTalon.setInverted(robotConstants.shooterConstants.IS_RIGHT_MOTOR_INVERTED);
+        rightTalon.setSensorPhase(robotConstants.shooterConstants.IS_RIGHT_ENCODER_INVERTED);
+        resetEncoders();
+    }
+
+    /**
+     * @param power The power to set the talons in open loop. Value should be between -1.0 and 1.0.
+     */
+    @Override
+    public void move(double power) {
+        leftTalon.set(power);
+        rightTalon.set(power);
+    }
+
+    public void startPID() {
+        startPID(robotConstants.shooterConstants.DEFAULT_RPM);
+    }
+
+    /**
+     * Starts using velocity PID instead of open-loop.
+     *
+     * @param velocitySetpoint velocity to set the talons.
+     */
+    public void startPID(double velocitySetpoint) {
+        double velocityInTalonUnits = velocitySetpoint * robotConstants.shooterConstants.LEFT_UNITS_PER_ROTATION
+                / 600;
+        leftTalon.set(TalonFXControlMode.Velocity, velocityInTalonUnits);
+        rightTalon.set(TalonFXControlMode.Velocity, velocityInTalonUnits);
+    }
+
+
+    public void enableTuning() {
+        isTuning = true;
+        // left shooter gains
+        SmartDashboard.putNumber("PID/LeftShooter/kP", 0);
+        SmartDashboard.putNumber("PID/LeftShooter/kI", 0);
+        SmartDashboard.putNumber("PID/LeftShooter/kD", 0);
+        SmartDashboard.putNumber("PID/LeftShooter/kF", 0);
```

Comment:
SmartDashboard.putNumber("PID/LeftShooter/kP", 0);
        SmartDashboard.putNumber("PID/LeftShooter/kI", 0);
        SmartDashboard.putNumber("PID/LeftShooter/kD", 0);
        SmartDashboard.putNumber("PID/LeftShooter/kF", 0);

use ShuffleboardLayout for left and right. see [https://docs.wpilib.org/en/latest/docs/software/wpilib-tools/shuffleboard/layouts-with-code/organizing-widgets.html](url)